communities: add "About communities" and "Manage" pages (#20)
CERNDocumentServer/docs-cds-rdm: docs/communities/communities.md

# Communities

> On CDS, communities are spaces in which records are organised and managed.

Reviewer: ```suggestion
On CDS, communities are spaces in which records are curated by specific groups of people, depending on community's scope.
```

> Depending on each community's settings, they can allow:

Reviewer: cosmetics: repetition of word "depending"

> - **Grouping related content** visually, giving users a focused browsing experience.

Reviewer: I would leave this out, since we don't really mean it for grouping of content

- **Curating** records; specific community members can review submitted records before publication.
- **Managing permissions** by specifying which users or CERN Groups can submit new records.
- **Creating custom pages** such as an About page or a Curation Policy

All records on CDS belong to at least one community.

> When a record is created, it must be submitted to an initial community, but after publication it can also be submitted

Reviewer 2: ```suggestion
When a record is created, it must be submitted to a community, but after publication it can also be submitted
```

## Create a community

> In order to ensure communities have a coordinated, user-friendly structure, we don't allow users to create communities directly.

Reviewer: please smoothen up "we don't allow" phrasing, as it was suggested earlier - it is too direct and it does not sound very polite

> In order to ensure communities have a co-ordinated, user-friendly structure, we don't allow self-service creation.

Reviewer 2: ```suggestion
In order to ensure communities have a co-ordinated, user-friendly structure, we don't allow users to directly create communities.
```

> In order to ensure communities have a co-ordinated, user-friendly structure, we don't allow self-service creation.

Reviewer: ```suggestion
In order to ensure communities have a coordinated, user-friendly structure, CDS team is creating the community on request, which will be carefully evaluated.
```

Please [contact the CDS team via ServiceNow](https://cern.service-now.com/service-portal?id=sc_cat_item&name=request&se=CDS-Service), specifying the following details:

- The intended name
- A square logo (maximum 1MB)
- A short description of what the community will be used for
- The type of records that will be published in the community (please send links to examples if possible)

> - Whether reviews should be mandatory for newly submitted records

Reviewer 2: link to the manage page

- Whether the community's contents should be publicly visible
- The name of a dedicated [GMS group](https://auth.docs.cern.ch/groups/overview/) to set as the initial member of the community

## Members and roles

Communities have members, each with a role which describes their level of access.

> A member can be a CERN user or a GMS group.

Reviewer 2: ```suggestion
A member can be a CERN user or group.
```

Community owners can specify the initial members, which can then later be edited.

Each member can have one of the following roles:

- **Reader**: Can view all records inside the community, including restricted ones.
- **Curator**: Can curate/review records submitted to the community and can view all records.
- **Manager**: Can manage members and roles, can curate records, and can view all records.
- **Owner**: Full administrative control over all aspects of the community.

> !!! info "Access for non-members"

Reviewer 2: ```suggestion
!!! info "Access for non-members of the community"
```

    CDS users who are not members of a community can still view its public records and its details.
    This can be changed in the community's settings.

    Non-members cannot view the community's restricted records, unless they have been granted [explicit permission](../deposit/access-share.md) on a per-record basis.

[Find out more](./manage.md#members) about managing community members.

## View your communities

> To view the communities you are a member, or to see a full list of communities on CDS, click on the "Communities" tab at the top of the page.

Reviewer 2: ```suggestion
To view the communities you are a member of, or to see a full list of communities on CDS, click on the "Communities" tab at the top of the page.
```
